lib/carrierwave/neo4j.rb

```diff
@@ -48,7 +48,7 @@ def clear_#{column}
 
         def _mounter(column)
           @_mounters ||= {}
-          @_mounters[column] ||= CarrierWave::Mount::Mounter.new(self, column)
+          @_mounters[column] ||= CarrierWave::Mounter.new(self, column)
```

Comment:
It looks like this was changed here:

https://github.com/carrierwaveuploader/carrierwave/commit/1ac1268d9bb7e02b16b0792b4bc755bc4e36fcda

The carrierwave gem seems to be encouraging people to use the `master` branch unless they're using Rails 3, but they `README` mentions something about a forthcoming 1.0 release.  Their most recent publication to rubygems was 0.10.0 which was published Feb 26th, 2014 and that commit is from Oct 16, 2014.

Perhaps this could be something like:

```ruby
mounter_class = defined?(CarrierWave::Mounter) ? CarrierWave::Mounter : CarrierWave::Mount::Mounter
@_mounters[column] ||= mounter_class.new(self, column)
```